pxtlib/sprite-editor/spriteEditor.ts

```diff
@@ -129,7 +129,7 @@ namespace pxtsprite {
             });
 
             this.sidebar = new SideBar(['url("#alpha-background")'].concat(this.colors), this, this.group);
-            this.sidebar.setColor(1);
+            this.sidebar.setColor(3);
```

Comment:
what happens if the sprite editor has less than 3 colors?

Reply:
Good call, fixed